nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -371,7 +372,7 @@ void Costmap2DROS::mapUpdateLoop(double frequency)
       layered_costmap_->getBounds(&x0, &xn, &y0, &yn);
       publisher_->updateBounds(x0, xn, y0, yn);
 
-      rclcpp::Time now = now();
+      rclcpp::Time now = this->now();
```

Comment:
this-> can be left out

Reply:
I tried to leave it out, but I specifically received a compilation error here saying there was no match.